src/participant/prompts/schema.ts

```diff
@@ -21,9 +22,10 @@ export class SchemaPrompt extends PromptBase<SchemaPromptArgs> {
     return `You are a senior engineer who describes the schema of documents in a MongoDB database.
 The schema is generated from a sample of documents in the user's collection.
 You must follow these rules.
-Rule 1: Try to be as concise as possible.
-Rule 2: Pay attention to the JSON schema.
-Rule 3: Mention the amount of documents sampled in your response.
+Rule 1: Your answer should always describe the schema of documents in the collection.
```

Comment:
This change seems to work; I tried to trick the Copilot to get confused by previous `/query` queries and it seems to be more consistent even with short prompts i.e. `/schema users`

Open to suggestions if you see some other way.